web/docs/integrations/zeek.md

```diff
@@ -132,29 +130,30 @@ tenzir:
              JSONStreaming::disable_default_logs=T
              JSONStreaming::enable_log_rotation=F
              json-streaming-logs"
-      | read zeek-json
+      | read_zeek_json
```

Comment:
This can't work, UDOs don't support TQL2 currently.

Reply:
Will it work though, say within a few months? Then it's better to keep it.

Reply:
I'll let @jachris decide that.

Reply:
With some of the recent changes, it should not be too hard to add a hacky version of UDOs. You'd need to add something like `// tql2` here (or use a different config section) so that we know what to do. Also, FWIW, the "proper" solution probably looks quite different (to enable parameter definitions etc.), and then we'll also want to change the docs. But we could do something close to what we have right now rather soon if needed.

Reply:
Great. I created a story and assigned it to you: https://github.com/tenzir/issues/issues/2352